direct/io-kafka/src/test/java/cz/o2/proxima/direct/kafka/LocalKafkaCommitLogDescriptorTest.java

```diff
@@ -1004,7 +1008,7 @@ public void onIdle(OnIdleContext context) {
     assertTrue(watermark.get() < now * 10);
   }
 
-  @Test(timeout = 10000)
+  @Test()
```

Comment:
```suggestion
  @Test
```

Reply:
I will return the timeout back. I removed it just for debugging.